test/connectors/getFinancialDetails/GetFinancialDetailsConnectorSpec.scala

```diff
@@ -52,86 +53,12 @@ class GetFinancialDetailsConnectorSpec extends SpecBase with LogCapturing {
     when(mockAppConfig.eisEnvironment).thenReturn("asdf")
     when(mockAppConfig.queryParametersForGetFinancialDetail(Matchers.any(), Matchers.any())).thenReturn("?foo=bar")
   }
-  
-  val mockGetFinancialDetailsModelAPI1811: GetFinancialDetails = GetFinancialDetails(
-    documentDetails = Seq(DocumentDetails(
-      documentId = "1234",
-      accruingInterestAmount = None,
-      interestOutstandingAmount = None,
-      metadata = DocumentDetailsMetadata(
-        taxYear = "2022",
-        documentDate = LocalDate.now,
-        documentText = "asdf",
-        documentDueDate = LocalDate.now,
-        documentDescription = None,
-        formBundleNumber = None,
-        totalAmount = 0.00,
-        documentOutstandingAmount = 0.00,
-        lastClearingDate = None,
-        lastClearingReason = None,
-        lastClearedAmount = None,
-        statisticalFlag = true,
-        informationCode = None,
-        paymentLot = None,
-        paymentLotItem = None,
-        interestRate = None,
-        interestFromDate = None,
-        interestEndDate = None,
-        latePaymentInterestID = None,
-        latePaymentInterestAmount = None,
-        lpiWithDunningBlock = None,
-        accruingPenaltyLPP1 = None
-      ))
-    ),
-    financialDetails = Seq(FinancialDetails(
-      documentId = "asdf",
-      taxPeriodFrom = None,
-      taxPeriodTo = None,
-      items = Seq(FinancialItem(
-        dueDate = None, clearingDate = None, metadata = FinancialItemMetadata(
-          subItem = None,
-          amount = None,
-          clearingReason = None,
-          outgoingPaymentMethod = None,
-          paymentLock = None,
-          clearingLock = None,
-          interestLock = None,
-          dunningLock = None,
-          returnFlag = None,
-          paymentReference = None,
-          paymentAmount = None,
-          paymentMethod = None,
-          paymentLot = None,
-          paymentLotItem = None,
-          clearingSAPDocument = None,
-          codingInitiationDate = None,
-          statisticalDocument = None,
-          DDCollectionInProgress = None,
-          returnReason = None,
-          promisetoPay = None
-        )
-      )),
-      originalAmount = None,
-      outstandingAmount = None,
-      chargeReference = None,
-      mainTransaction = None,
-      metadata = FinancialDetailsMetadata(
-        taxYear = "2022",
-        chargeType = None,
-        mainType = None,
-        periodKey = None,
-        periodKeyDescription = None,
-        businessPartner = None,
-        contractAccountCategory = None,
-        contractAccount = None,
-        contractObjectType = None,
-        contractObject = None,
-        sapDocumentNumber = None,
-        sapDocumentNumberItem = None,
-        subTransaction = None,
-        clearedAmount = None,
-        accruedInterest = None
-      )
+
+  val mockGetFinancialDetailsModelAPI1811: FinancialDetails = getFinancialDetails.FinancialDetails(
```

Comment:
Do you need the getFinancialDetails prefix?

Reply:
Removed ✅